### ISSUE TITLE
Reduce code complexity by piggybacking the "welcome" check based on whether or not the "new contributor" indicator is visible

### ISSUE THREAD

**Issue by gparyani** [Benjol/SE-AutoReviewComments#155]
Currently, the script bases whether or not to put in a welcome message in an automatic comment on its own criteria.

I think that the code's complexity can be reduced by not using its own criteria, but instead simply checking to see if the "new contributor" indicator is visible on that specific post. I usually manually type in the welcome message myself if the indicator is present, but the script didn't add it.

**Comment by Benjol** [comment]
Good idea. Don't have time myself, but if someone wants to have a look, I'll pull.
